src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -257,21 +282,21 @@ private void validateMedicalInventory(MedicalInventory medInventory) throws OHDa
 	 *
 	 * @param inventory the {@link MedicalInventory}
 	 * @param inventoryRowSearchList- The list of {@link MedicalInventory}
+	 * @param allMedicals if {@code true} will check also medicals not in {@code inventoryRowSearchList}
 	 * @throws OHServiceException
 	 */
-	public void validateMedicalInventoryRow(MedicalInventory inventory, List<MedicalInventoryRow> inventoryRowSearchList)
+	public void validateMedicalInventoryRow(MedicalInventory inventory, List<MedicalInventoryRow> inventoryRowSearchList, boolean allMedicals)
 		throws OHServiceException {
 		LocalDateTime movFrom = inventory.getInventoryDate();
 		LocalDateTime movTo = TimeTools.getNow();
-		StringBuilder medDescriptionForLotUpdated = new StringBuilder("\n"); // initial new line
-		StringBuilder medDescriptionForNewLot = new StringBuilder("\n"); // initial new line
-		StringBuilder medDescriptionForNewMedical = new StringBuilder("\n"); // initial new line
+		StringBuilder medDescriptionForLotUpdated = new StringBuilder('\n'); // initial new line
```

Comment:
```suggestion
		StringBuilder medDescriptionForLotUpdated = new StringBuilder("\n"); // initial new line
```
I was wrong; `StringBuilder` takes a `string` argument and not a `char`.